bigchaindb/pipelines/block.py

```diff
@@ -1,6 +1,6 @@
 """This module takes care of all the logic related to block creation.
```

Comment:
Agree with @TimDaub this was confusing to begin with (when I worked on it) and now that blocks are objects, the distinction is necessary